### ISSUE TITLE
[Adaptive Cards> Flight Itinerary.json]: Screen Reader focus is going on the hidden place while navigating the 'Flight Itinerary' card.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8979]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0
Screen Reader: Narrator

**Pre-Requisite:**
Turn on the Narrator and turn on the scan mode.

**Repro Steps:**
1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to 'Flight Itinerary.Json' button by using arrow key and activate it.
4. Flight Itinerary.Json adaptive card will appear on right sight of the pane.
5. Navigate to Flight Itinerary.Json card by using arrow key.
6. Press down arrow key and navigate the screen and observe the issue.

**Actual Result:**
While navigating the Flight Itinerary Cards screen, After Tue, May 30, 2017 12:25 PM screen reader focus is landing on the hidden element and it is announcing as 'Departs from' and on second hidden place it is announcing as 'Arrives at'.

**Expected Result:**
Screen Reader focus should not lands on the hidden place while navigating the Flight Itinerary card. Either it should not lands on the hidden place or visually label should be appeared.

**Note:**
Same issue is repro'ing with NVDA.

**User Impact:**
It will impact the screen reader user as they will get confused if the focus lands on the hidden place.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/78d13567-9de4-44fe-841e-f3460f3dd184

![While navigating in Scan mode, Screen Reader focus is landing on the hidden place](https://github.com/user-attachments/assets/cc8a6a3d-fb12-4e63-a684-aafc609119ae)

_No response_

### Card JSON

```JSON
NA
```
